### ISSUE TITLE
Remove myokit.UnsupportedFunction

### ISSUE THREAD

**Issue by MichaelClerx** [MichaelClerx/myokit#440]
Exists solely for imports, but no longer sure that's a good approach

**Comment by MichaelClerx** [comment]
Current uses:

```./myokit/tests/test_formats_sympy.py:
        u = myokit.UnsupportedFunction('frog', x)```
Tests if it raises an exception

```./myokit/tests/test_formats_mathml_content.py:```
Tests if it raises an exception

```
./myokit/tests/test_formats_expression_writers.py:
        u = myokit.UnsupportedFunction('frog', x)
```
Tests if it raises an exception

**Comment by MichaelClerx** [comment]
Closed in #439
